changelog.md

```diff
@@ -45,6 +45,7 @@
 - A bug allowed `macro foo(): int = 123` to compile even though a
   macros has to return a `NimNode`. This has been fixed.
 
+- With the exception of `uint` and `uint64`, conversion to unsigned types are now range checked during runtime.
```

Comment:
This seems to be outdated now? Where is the special casing for ``uint`` and ``uint64``?

Reply:
Special casing: https://github.com/nim-lang/Nim/blob/6a94599d66484396b4e080e5bb8c1aa8e8f129af/compiler/transf.nim#L469-L475

The issue is that the runtime checks doesn't support `uint`/`uint64`.